sdk/keyvault/keyvault-keys/src/generated/models/index.ts

```diff
@@ -9,6 +9,43 @@
 
 import * as coreHttp from "@azure/core-http";
 
+/**
+ * An interface representing KeyReleaseCondition.
+ */
+export interface KeyReleaseCondition {
```

Comment:
Per comment in our Teams channel, do not implement any of the secure key release models. The design is changing now.

Reply:
This is the generated code though 🤔 Should I exclude this from the generated code?

Reply:
For now if you want to make headway on the other changes, probably a good idea. If you can, maybe use a transform to turn `release_policy` properties into a `string`, i.e. the `KeyReleasePolicy` model effectively becomes just an opaque string. This is the likely change for 7.2-preview coming soon.

Reply:
We can wait for that to appear in the swagger! I mean, this generated code doesn't appear in the public API. Would that be ok?

Reply:
If this is all generated and not exported, that should be fine.